### DISCUSSION TITLE
How to get message object from Slash command interaction

### DISCUSSION THREAD

**Discussion by alvinbengeorge** [nextcord/nextcord#422] [Q&A] [answered]
Ok, so basically I have interaction, and when I checked about the message, that is `interaction.message` it returns NoneType
```py
['__bool__', '__class__', '__delattr__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__']
```
Anyone knows how to get the message object so that I can pass it to a function which does pagination

**Comment by Skelmis** [comment, marked as answer]
If we go check the documentation, linked [here](https://nextcord.readthedocs.io/en/latest/api.html?highlight=interaction#nextcord.Interaction) we can see it has a method called [original_message](https://nextcord.readthedocs.io/en/latest/api.html?highlight=interaction#nextcord.Interaction.original_message) which says it should fetch and return the interaction response message that triggered the interaction.

If ``.message`` is ``None`` you can try use that

**Reply by alvinbengeorge** [reply]
Should check that out, I'll reply as soon as I see that, I did check it out, but had issues from what I remember

**Reply by alvinbengeorge** [reply]
Also is NoneType, I have server intent and message intent
so..... and it works with other command also

**Comment by ooliver1** [comment]
Slash commands are not message based, the message attribute is for views (buttons and selects) and original_message is the response

**Reply by alvinbengeorge** [reply]
Also gave none type for original_message and also I was thinking of reusing the regular functions but it has no attribute author and stuff, which is fine

**Reply by alvinbengeorge** [reply]
How do I add reactions then?
